### PR TITLE
updates inline comments 

### LINKED ISSUE #234: update inline comments
just noticed these comments were off for `GET` user requests

https://github.com/DoSomething/northstar/blob/67ea51f5fba7fb777c3117c8a3326c8686b36194/app/Http/Controllers/UserController.php#L27

@chloealee @DFurnes can you take a look at these comments and others when working in NS

### PR DESCRIPTION
**Pull request by chloealee**
#### What does this PR do?
Fixes comment in UserController.php construct function to have GET all capitalized. Updates comment in AvatarController.php store function to remain consistent with other endpoints. 

#### What are the relevant tickets?
Fixes #234

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -23,8 +23,7 @@ public function __construct(Phoenix $phoenix)
 
     /**
      * Display a listing of the resource.
-     * GET /users
-     * Get /users?attr1=value1&attr2=value2&...
+     * GET /users/:term/:identifier
```
Comment (DFurnes) [inline app/Http/Controllers/UserController.php]: Wait, this is the index method... I think `/users/:term/:identifier` is for the `show` method below.

Comment (DFurnes) [inline app/Http/Controllers/UserController.php]: I'm pretty sure the two examples here are correct, although `Get` should be all-caps.

Comment (chloealee) [inline app/Http/Controllers/UserController.php]: ohhhh I see now. And the second just gives an example of how to use the queries? Changed!

**Comment by chloealee** [comment]
@DFurnes / @angaither whenever you get a sec to take a look!

**Comment by DFurnes** [comment]
Looks good to me! :+1:

**Pull request merged**